webapp/src/components/todo_item/todo_item.jsx

```diff
@@ -27,7 +27,17 @@ function TodoItem(props) {
     const [editTodo, setEditTodo] = useState(false);
     const [message, setMessage] = useState(issue.message);
     const [description, setDescription] = useState(issue.description);
+    const MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
     const [hidden, setHidden] = useState(false);
+    const date = new Date(issue.create_at);
+    const year = date.getFullYear();
+    const month = MONTHS[date.getMonth()];
+    const day = date.getDate();
+    const hours = date.getHours();
+    const minutes = '0' + date.getMinutes();
+    const seconds = '0' + date.getSeconds();
+    const formattedTime = hours + ':' + minutes.substr(-2) + ':' + seconds.substr(-2);
+    const formattedDate = month + ' ' + day + ', ' + year;
```

Comment:
This should be way easier with `toLocaleDateString` and `toLocaleTimeString`. You can play in the web browser console to get the exact thing you want. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

You should be able to get the current locale either from the user, or the intl object. Not sure if something similar is already also in the intl object.